src/main/java/net/sf/jabref/gui/mergeentries/FetchAndMergeEntry.java

```diff
@@ -58,8 +58,9 @@ public FetchAndMergeEntry(BibEntry entry, BasePanel panel, List<String> fields)
 
                     try {
                         fetchedEntry = new IsbnFetcher().performSearchById(fieldContent.get());
-                    } catch (FetcherException fe) {
-                        fe.printStackTrace();
+                    } catch (FetcherException e) {
+                        panel.frame().setStatus(
+                                Localization.lang("Cannot get info based on given %0:_%1", type, fieldContent.get()));
```

Comment:
Please log the exception, too, e.g. Logger.error/debug (just look at other code pieces :smile:  )
And I think the underscore in the Localization is wrong, should be just a space.